pkg/types/build.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+//BuildOptions define the options available for build
+type BuildOptions struct {
+	BuildArgs     []string
+	CacheFrom     []string
+	File          string
+	NoCache       bool
+	OutputMode    string
+	Path          string
+	Secrets       []string
+	Tag           string
+	Target        string
+	Namespace     string
+	BuildToGlobal bool
+	Args          []string
```

Comment:
minor thing, can we add here some comments regarding the meaning of each Option, i find Args and BuildArgs very similar and maybe this can be confusing in the future

Reply:
+1, I'm always get confused by them

Reply:
👍 
I will add a comment and rename `options.Args` into `options.CommandArgs` so it's clearer what those variables refers to.